bigtable/admin.go

```diff
@@ -294,6 +313,17 @@ func (ac *AdminClient) CreateTableFromConf(ctx context.Context, conf *TableConf)
 		tbl.ChangeStreamConfig = &btapb.ChangeStreamConfig{}
 		tbl.ChangeStreamConfig.RetentionPeriod = durationpb.New(conf.ChangeStreamRetention.(time.Duration))
 	}
+	if conf.AutomatedBackupPolicy != nil && *conf.AutomatedBackupPolicy != DisableAutomatedBackupPolicy {
+		// Create Automated Backup Policy
+		if (conf.AutomatedBackupPolicy.Frequency == nil && conf.AutomatedBackupPolicy.RetentionPeriod != nil) ||
+			(conf.AutomatedBackupPolicy.Frequency != nil && conf.AutomatedBackupPolicy.RetentionPeriod == nil) {
```

Comment:
I think a more idiomatic way to express xor is (A || B) && !(A && B)

Reply:
Thanks! Updated

Reply:
Thanks. Updated